requirements/tox_requirements27.txt

```diff
@@ -1,9 +1,9 @@
 mock==3.0.5
-requests_oauthlib==1.2.0
+requests_oauthlib==1.3.0
 cryptography==2.8
 ddt==1.2.2
-django==1.11.24
-django-oauth-toolkit==1.0.0  # not needed in production, unless you need to create provider's Applications
+django==1.11.17
+django-oauth-toolkit==1.1.3  # not needed in production, unless you need to create provider's Applications
```

Comment:
So we have to downgrade Django? Is this a Python 2 thing?

Reply:
We don't necessarily have to, but `djano1.11.17` is what is used currently in LTI. And as the LTI is probably the most important python2 user of this code, I was thinking it would be nice to test it with its django version.